doc/examples/transform/plot_tps_deformation.py

```diff
@@ -61,14 +61,14 @@
 
 labels = ['1', '2', '3', '4', '5', '6']  # Adjust the number of labels to match the number of points
 
-axs[0].imshow(astronaut[..., ::-1], origin='upper', cmap='gray')
+axs[0].imshow(astronaut, cmap='gray')
```

Comment:
Nice! Looks a lot better now. Maybe we can make the markers more visible? I think removing or shrinking the figure size should do it. Essentially the figure is generated for a larger area but is shrunk when it is embedded into the example which makes the font to small to read...

Reply:
Another approach is to set marker size with`s` parameter and annotation font size with 
`font = {'size': 15}` 
`plt.rc('font', **font)`. 
However, your suggestion is simpler and does the job. 👍